plugins/modules/virt.py

```diff
@@ -575,7 +582,7 @@ def main():
     module = AnsibleModule(
         argument_spec=dict(
             name=dict(type='str', aliases=['guest']),
-            state=dict(type='str', choices=['destroyed', 'pause', 'running', 'shutdown']),
+            state=dict(type='str', choices=['destroyed', 'paused', 'running', 'shutdown']),
```

Comment:
According to https://docs.ansible.com/ansible/latest/modules/virt_module.html it's 'pause'. If we switch to 'paused' then we'll need an alias for backwards compatibility.

Reply:
I think you confused it with command, which has a 'pause' command... this is state, where the documentation was correct but the argument_spec was incorrect